js/app/controllers/settingscontroller.js

```diff
@@ -48,6 +48,15 @@ angular.module('Tasks').controller('SettingsController', [
 						name: t('tasks', 'Automatic')
 					}
 				];
+				this._$scope.parsingOptions = [
```

Comment:
I think the changes to this file are not used, since the value if the checklists tasks should be shown are stored per task.